gtclang/src/gtclang/Frontend/GTClangPreprocessorAction.cpp

```diff
@@ -877,9 +901,9 @@ class GTClangLexer {
 
   /// @brief Get the replacements
   const std::vector<clang::tooling::Replacement>& getReplacements() const { return replacements_; }
-};
+}; // namespace
```

Comment:
this is a leftover from a parenthesis mismatch

Reply:
Thanks for catching!